src/include/generation.h

```diff
@@ -0,0 +1,34 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * WT_GENERATION_COOKIE --
+ *   State passed through to callbacks during the session walk logic when
+ *   tracking a generation across all sessions.
+ */
+struct __wt_generation_cookie {
+    bool active;
```

Comment:
We discussed this, but writing it down so we don't forget - a clearer distinction between in/out fields on these structs would be good.

Reply:
This one ended up being kinda funky, worth an extra look at the commit: d6e228424

Reply:
That looks like a straightforward renaming? Is there something I missed?

Reply:
`ret_oldest_gen` is one example where it is both a return value and value utilized across the callbacks so I didn't have a good intuition about whether it really should be tagged with `ret_`.

Reply:
Ah OK. I think it's fine as-is.